src/Contract/Operation/Firstable.php

```diff
@@ -24,7 +24,7 @@ interface Firstable
      *
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#first
      *
-     * @return Collection<TKey, T>
+     * @return T
```

Comment:
```suggestion
     * @return T|null
```

`first` can return _null_

Reply:
Fixed, fixuped, rebased.